vendor.conf

```diff
@@ -15,7 +15,7 @@ github.com/containerd/cgroups 4994991857f9b0ae8dc439551e8bebdbb4bf66c1
 github.com/containerd/continuity 004b46473808b3e7a4a3049c20e4376c91eb966d
 github.com/containernetworking/cni v0.7.0-rc2
 github.com/containernetworking/plugins v0.7.4
-github.com/containers/image v2.0.0
+github.com/containers/image podman-1.4.2
```

Comment:
This can't be right is it?
Tests are all kinds of unhappy too.

Reply:
I hope so. The `podman-1.4.2` branch in c/image was created for this version of Podman.

Reply:
@cevich @mheon do you know what's going with Cirrus in this branch?

Reply:
Cirrus here is unfortunately trying to test using the same images as master, which don't work with the old cirrus configs lying around - I guess we never resolved this when it happened last time... Damn.

Reply:
I think we could just merge when travis passes?

Reply:
There is no travis - we have absolutely no testing on this branch at the moment

Reply:
Can we bump the priority to revive CI on this branch?

Reply:
Let's talk at scrum - it may be something relatively simple.